doc/rst/language/spec/ranges.rst

```diff
@@ -58,55 +61,39 @@ follows.
 
 -  The *alignment* is either a specific index value or is *ambiguous*.
    It defines how the represented sequence’s members are aligned
-   relative to the stride. For a range with a stride other than 1 or -1,
-   ambiguous alignment means that the represented sequence is undefined.
-   In such a case, certain operations discussed later result in an
-   error.
-   The alignment is always between zero and :math:`|stride|-1`,
-   inclusively.
-
-More formally, the represented sequence for the range
-:math:`(low, high, stride, alignmt)` contains all indices :math:`ix`
-such that:
-
-=========================================================================== ============================================
-:math:`low \leq ix \leq high`                                               if :math:`stride = 1` or :math:`stride = -1`
-:math:`low \leq ix \leq high` and :math:`ix \equiv alignmt \pmod{|stride|}` if :math:`alignmt` is not ambiguous
-the represented sequence is undefined                                       otherwise
-=========================================================================== ============================================
-
-The sequence, if defined, is increasing if :math:`stride > 0` and
-decreasing if :math:`stride < 0`.
+   relative to the stride. The alignment, when unambiguous,
+   is always between zero and :math:`|stride|-1`, inclusively.
+
+A range with unambiguous alignment is called *aligned*. Otherwise
+the range is *unaligned* and its represented sequence is undefined.
+
+The represented sequence for an aligned range
+:math:`(low, high, stride, alignmt)`
```

Comment:
This is probably out of scope, but I think when using a multi-letter variable name in LaTex, you're supposed to use `\text{low}` (e.g.) instead of just `low`; otherwise, it's typeset as the implicit multiplication of three characters. `\textit{low}` could be used instead to make it italicized. The kerning is all wrong otherwise.

Reply:
Let us take this into a separate PR.